docs/core/project-sdk/overview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: .NET Core project SDK overview
+description: Learn about the .NET Core project SDKs.
+ms.date: 02/02/2020
+ms.topic: conceptual
+---
+# .NET Core project SDKs
+
+.NET Core projects are associated with a software development kit (SDK). Each project SDK is a set of MSBuild [targets](/visualstudio/msbuild/msbuild-targets) and associated [tasks](/visualstudio/msbuild/msbuild-tasks) that are responsible for compiling, packing, and publishing code.
+
+## Available SDKs
+
+The following SDKs are available for .NET Core:
+
+| ID | Description | Repo|
+| - | - | - |
+| `Microsoft.NET.Sdk` | The .NET Core SDK | https://github.com/dotnet/sdk |
+| `Microsoft.NET.Sdk.Web` | The .NET Core [Web SDK](/aspnet/core/razor-pages/web-sdk) | https://github.com/aspnet/websdk |
+| `Microsoft.NET.Sdk.Razor` | The .NET Core [Razor SDK](/aspnet/core/razor-pages/sdk) |
+| `Microsoft.NET.Sdk.Worker` | The .NET Core Worker Service SDK |
+| `Microsoft.NET.Sdk.WindowsDesktop` | The .NET Core WinForms and WPF SDK |
+
+The .NET Core SDK is the base SDK for .NET Core. The other SDKs depend on the .NET Core SDK. For example, the Web SDK depends on both the .NET Core SDK and the Razor SDK.
+
+## Project files
```

Comment:
This section should mention how to specify SDK's that come from NuGet `MSBuild.Sdk.Extras/2.0.54` or by specifying the SDK and version in the `global.json`, only the short name `MSBuild.Sdk.Extras` is needed.